bloom-housing/bloom
# feat: core changes for application style alignment

# Pull Request Template

## Issue Overview

[This PR addresses #issue](https://github.com/metrotranscom/doorway/issues/364)



## Description

From the linked Figma, this PR adds the styles that were indicated as also being core styles, and uptakes seeds in some places in the application to support overriding when we pull over to Doorway.

* Uptake of seeds Card component in the application
* Removing placeholder text from fields
* Removing caps styling from fields
* Standardizing text placement and header styling across different questions

I created a share-able layout for the structure of an application page as that was duplicated across all pages.

## How Can This Be Tested/Reviewed?

Compare the application on this branch to the Figma and/or to dev, there's lots of great documentation in there about what is a core change but there were lots of small changes so feel free to reach out if you're not sure if something is intentional!

Ensure that each page has the right new styles and that you can navigate forward / backward as in dev - [this doc](https://docs.google.com/document/d/1-ENOXNxGsYLI4TsCvKeO7nGz6FCiQsCrkjcxW19k6jU/edit?usp=sharing) has screenshots of each page compared to dev with notes.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
